invenio_stats/contrib/file_download/file_download_v1.json

```diff
@@ -34,11 +34,12 @@
         "visitor_id": {
           "type": "string",
           "index": "not_analyzed"
+        },
+        "collection": {
+          "type": "string",
+          "index": "not_analyzed"
         }
       }
     }
-  },
-  "aliases": {
-    "events-filedownload": {}
```

Comment:
However it should be named `events-stats_file_download` just to be consistent with the above template name